### ISSUE TITLE
deploy.ps1 is failing because the parameter name 'f' is ambiguous

### ISSUE THREAD

**Issue by avidigmi** [microsoft/botframework-solutions#3445]
#### What project is affected?
Virtual Assistant Template deployment

#### What language is this in?
C#

#### What happens?
The execution of `./Deployment/Scripts/deploy.ps1` fails with this error:

```ps
Line |
 213 |  $timestamp = Get-Date -f MMddyyyyHHmmss
     |                        ~~
     | Parameter cannot be processed because the parameter name 'f' is ambiguous. Possible
     | matches include: -Format -FromUnixTime.
```

#### What are the steps to reproduce this issue?
Run `./Deployment/Scripts/deploy.ps1`

**Comment by darrenj** [comment]
Fixed in #3429 last week, available in next branch until point release this Friday.

**Comment by avidigmi** [comment]
Thanks!
